src/commands/install.yml

```diff
@@ -6,13 +6,21 @@ parameters:
     type: "string"
     description: "Specify version of terraform to install."
     default: "0.13.5"
+  os:
+    type: string
```

Comment:
If we have a set of known values, let's use the enum type.
https://circleci.com/docs/2.0/reusing-config/#enum

Reply:
done